src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -1,21 +1,24 @@
 package com.wikia.webdriver.testcases.chattests;
 
-import java.util.List;
-
-import org.testng.annotations.Test;
-
+import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.DontRun;
-import com.wikia.webdriver.common.core.annotations.RelatedIssue;
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.properties.Credentials;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.chatpageobject.ChatPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.chatpageobject.ChatPage;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialVersionPage;
+import org.testng.annotations.Test;
+
+import java.util.List;
 
 public class ChatTests extends NewTestTemplate {
 
+  private String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
+  private String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "MESSAGE_ON_PRIVATE_CHAT" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)